test/invariants/fuzz/Greeter.t.sol

```diff
@@ -2,36 +2,39 @@
 pragma solidity 0.8.23;
 
 import {Greeter, IERC20} from 'contracts/Greeter.sol';
+import {CommonBase} from 'forge-std/Base.sol';
 
-interface IHevm {
-  function prank(address) external;
-}
-
-contract InvariantGreeter {
-  IHevm internal _hevm = IHevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
-
+contract InvariantGreeter is CommonBase {
```

Comment:
I suggest we follow the usual structure with properties and handler contracts in this repo. WDYT?

Reply:
It's reasonable, I wasnt particularly sold on any of the options. Do you want to keep all contracts in a single file in order for it to be easier to grasp in one look? or should I go all in on boilerplate and replicate the directory structure as well?